types/common.d.ts

```diff
@@ -18,7 +18,7 @@ export type Options = AnyObject<any>;
 /**
  * Type alias for Node.js callback functions
  */
-export type Callback<T = any> = (err: any, result?: T) => void;
+export type Callback<T = any> = (err?: Error | null, result?: T) => void;
```

Comment:
Enable the following usage patterns:

- `callback()`
- `callback(null)`

Reply:
Shouldn't it be `err?: any` - AFAIK normally any value passed to first parameter that's not null / undefined means an error. So ... for example, I can pass a string into the Callback?

Reply:
Node.js does allow the error parameter to be `any`.